system-parachains/asset-hubs/asset-hub-polkadot/src/lib.rs

```diff
@@ -1019,8 +1028,8 @@ pub type Block = generic::Block<Header, UncheckedExtrinsic>;
 pub type SignedBlock = generic::SignedBlock<Block>;
 /// BlockId type as expected by this runtime.
 pub type BlockId = generic::BlockId<Block>;
-/// The SignedExtension to the basic transaction logic.
-pub type SignedExtra = (
+/// The `TransactionExtension` to the basic transaction logic.
+pub type TxExtension = (
```

Comment:
(we can also add `ReclaimWeight` at the end of the pipeline)

Reply:
I noticed `WeightReclaim` was introduced in `frame-system@40.1.0` (`stable2503`). So, I'll be skipping this for now, because we're a bit time-strechted to release this upgrade.